mlflow/store/db_migrations/versions/3500859a5d39_add_model_aliases_table.py

```diff
@@ -0,0 +1,36 @@
+"""Add Model Aliases table
+
+Revision ID: 3500859a5d39
+Revises: 97727af70f4d
+Create Date: 2023-03-09 15:33:54.951736
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from mlflow.store.model_registry.dbmodels.models import SqlRegisteredModelAlias
+
+# revision identifiers, used by Alembic.
+revision = "3500859a5d39"
+down_revision = "97727af70f4d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        SqlRegisteredModelAlias.__tablename__,
+        sa.Column("name", sa.String(length=256), primary_key=True, nullable=False),
+        sa.Column("alias", sa.String(length=250), primary_key=True, nullable=False),
```

Comment:
```suggestion
        sa.Column("name", sa.String(length=256), nullable=False),
        sa.Column("alias", sa.String(length=250), nullable=False),
```

I think we can remove these because there is `registered_model_alias_pk`.